benchmarks/dynamo/common.py

```diff
@@ -1751,7 +1751,13 @@ def run(runner, args, original_dir=None):
     elif args.nothing:
         pass
     elif args.backend:
-        optimize_ctx = torch._dynamo.optimize(args.backend, nopython=args.nopython)
+        if args.backend == "ipex":
+            if args.amp:
+                optimize_ctx = torch._dynamo.optimize(backends.ipex_bf16)
```

Comment:
```suggestion
                optimize_ctx = torch._dynamo.optimize(backends.ipex_bf16, nopython=args.nopython)
```

Reply:
Is `ipex_bf16` actually doing the same thing as AMP?  I am wondering if this should be triggered by `--float16` rather than `--amp`.